docs-api/frontity-packages/features-packages/wp-source.md

```diff
@@ -641,18 +673,19 @@ Request entity from the WordPress REST API.
 
 **Arguments**
 
-| Name | Type | Required | Description |
-| :--- | :--- | :--- | :--- |
-| _`options`_ | object | yes | options object |
-| _`options`_.**`endpoint`** | string | yes | Name of the endpoint if is a `/wp/v2` endpoint \(e.g. `posts`\), or the full path of other REST endpoints \(e.g. `/acf/v3/posts`\). |
-| _`options`_.`params` | object | no | Any parameter that will be included in the query params. |
-| _`options`_.`api` | string | no | Overrides the value set with `api.set.` |
-| _`options`_.`isWpCom` | boolean | no | Overrides the value set with `api.set.` |
+| Name                       | Type    | Required | Description                                                                                                                                  |
+| :------------------------- | :------ | :------- | :------------------------------------------------------------------------------------------------------------------------------------------- |
+| _`options`_                | object  | yes      | options object                                                                                                                               |
+| _`options`_.**`endpoint`** | string  | yes      | Name of the endpoint if is a `/wp/v2` endpoint \(e.g. `posts`\), or the full path of other REST endpoints \(e.g. `/acf/v3/posts`\).          |
+| _`options`_.`params`       | object  | no       | Any parameter that will be included in the query params.                                                                                     |
+| _`options`_.`auth`         | string  | no       | Allows the Authorization header on the fetch() request to be set. Can use the value at [`state.source.auth`](wp-source.md#state-source-auth) |
```

Comment:
Maybe it would be better to say something more specific like: If not specified, will use the value from `state.source.auth` if that value is present in the state.